include/env.bash

```diff
@@ -77,6 +77,8 @@ CB_DB_ENV_DB - Name of the Cloudbreak database
 CB_DB_ENV_PASS - Password for the Cloudbreak database authentication
 CB_DB_ENV_SCHEMA - Used schema in the Cloudbreak database
 CB_DB_ENV_USER - User for the Cloudbreak database authentication
+CB_DB_ENV_SSL - enable of ssl certification or not
+CB_DB_ENV_CERT_FILE - the name of the cert file. This file should be under the "/certs" directory of the container. The value is "$CBD_CERT_ROOT_PATH:/certs" by default.
```

Comment:
The directory should be `$DEPLOYMENT_DIR/certs` where the certs should be placed in.